src/Admin/Routing/RouteAdmin.php

```diff
@@ -128,17 +131,18 @@ public function getExportFormats()
     protected function configureFieldsForDefaults($dynamicDefaults)
     {
         $defaults = array(
-            '_controller' => array('_controller', TextType::class, array('required' => false)),
-            '_template' => array('_template', TextType::class, array('required' => false)),
-            'type' => array('type', RouteTypeType::class, array(
+            '_controller' => ['_controller', TextType::class, ['required' => false, 'translation_domain' => 'CmfSonataAdminIntegrationBundle']],
+            '_template' => ['_template', TextType::class, ['required' => false, 'translation_domain' => 'CmfSonataAdminIntegrationBundle']],
```

Comment:
this is no admin extension, why specify the translation domain on a per field level?